packages/core/botpress/src/server/secured.js

```diff
@@ -79,6 +79,10 @@ module.exports = (bp, app) => {
     res.send(!util.isDeveloping)
   })
 
+  app.get('/api/community/hero', (req, res) => {
+    res.send(bp.botfile.hideHeroSection)
```

Comment:
I'd prefer if you wrapped the config in an object as the API isn't really descriptive of what the value is and doesn't scale to more props.

e.g.

```js
{ hidden: true }
```